packages/pangea-node-sdk/src/utils/crypto.ts

```diff
@@ -47,3 +50,131 @@ export function asymmetricDecrypt(
     )
     .toString("utf8");
 }
+
+/**
+ * Decrypt cipher text using the given asymmetric private key.
+ * @param privateKey Asymmetric private key.
+ * @param cipherText Cipher text.
+ * @param oaepHash Hash function to use for OAEP padding and MGF1.
+ * @param padding Padding value.
+ * @returns Decrypted text.
+ */
```

Comment:
copy-paste from `asymmetricDecrypt()` above?

Reply:
Yes. Skip export now and remove this wrong docs